<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCRemoveItemsForm</class>
 <widget class="QDialog" name="GCRemoveItemsForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Items from Profile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/gcresources.qrc">
    <normaloff>:/resources/goblinicon.jpg</normaloff>:/resources/goblinicon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="elementLayout">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="indentation">
          <number>12</number>
         </property>
         <column>
          <property name="text">
           <string>Element Hierarchy</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deleteElementLayout">
         <item>
          <spacer name="elementSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="removeChildButton">
           <property name="text">
            <string>Remove Child</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteElementButton">
           <property name="text">
            <string>Delete Element</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="elementHelpButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/gcresources.qrc">
             <normaloff>:/resources/helpcursor.png</normaloff>:/resources/helpcursor.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="attributeLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deleteAttributeLayout">
         <item>
          <spacer name="attributeSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="updateValuesButton">
           <property name="text">
            <string>Update Values</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteAttributeButton">
           <property name="text">
            <string>Delete Attribute</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="attributeHelpButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/gcresources.qrc">
             <normaloff>:/resources/helpcursor.png</normaloff>:/resources/helpcursor.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/gcresources.qrc"/>
 </resources>
 <connections/>
</ui>
